.gitignore

```diff
@@ -7,3 +7,6 @@
 src/aiokatcp.egg-info
 *.pyc
 __pycache__
+
+# macOS detritus
+.DS_Store
```

Comment:
I feel like that should go into your global `core.excludesfile` rather than here, because it's specific to your development environment rather than specific to the repository.

Reply:
Thanks, I did not know about that file... I merely cargo-culted some other repo's ignores (katpoint, etc). Much better idea!